paddle/framework/pybind.cc

```diff
@@ -200,6 +200,10 @@ All parameter, weight, gradient are variables in Paddle.
     return OpRegistry::CreateOp(desc);
   });
 
+  operator_base.def_static("support_gpu", [](const std::string &type) {
```

Comment:
`operator_base.def_static("support_gpu", &OpRegistry::SupportGPU);` should work.